src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -35,7 +35,7 @@ def process_file(config: MorphConfig, input_file: str | Path, output_file: str |
 
     lca_error = dialect_utils.check_for_unsupported_lca(source, sql, str(input_file))
     if lca_error:
-        parse_error_list.append(lca_error)
+        validate_error_list.append(lca_error)
```

Comment:
we are calling out as validation errors now?

Reply:
Since we are mainly checking the validity of query, thought of changing it to validation error. In case of parsing error, it will anyway get flagged by the transpiler. What do you think?

Reply:
make sense